test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -207,7 +207,7 @@ public void CanReadFileWithZeroTouchMigrationOfFunctionSignatureWithoutParameter
             string xmlstring =@"<Dynamo.Nodes.DSFunction guid=""f05953f3-6ead-44f7-b872-1e0203c784cc""
             type=""Dynamo.Nodes.DSFunction"" nickname=""DateTime.Now"" x=""259.5"" y=""260.5"" 
             isVisible=""true"" isUpstreamVisible=""true"" lacing=""Shortest""
-            isSelectedInput=""False"" assembly=""DSCoreNodes.dll"" function=""DSCore.DateTime.Now"" />";
+            isInput=""False"" assembly=""DSCoreNodes.dll"" function=""DSCore.DateTime.Now"" />";
```

Comment:
@ikeough I don't think this would cause the test here to fail, but was this part of a mass rename? It could be confusing if someone copies this xml later - it looks like you're still serializing to isSelectedInput - not isInput?